rep-2010.rst

```diff
@@ -234,7 +234,46 @@ Any existing tooling for aiding users in setting up ROS security should consider
 Command line interface
 ----------------------
 
-TODO: Consider proposing an interface for a CLI here. Although it seems a bit tangent to the core feature, I think it is important to the proposal since it makes it so people can immediately start using the feature.
+Suppose we have a service named ``/add_two_ints`` of type ``example_interfaces/srv/AddTwoInts``.
+We can have a tool like ``ros2 service echo /add_two_ints`` that internally subscribes to the two hidden topics ``example_interfaces/srv/AddTwoInts_RequestEvent``, ``example_interfaces/srv/AddTwoInts_ResponseEvent`` and echoes them. It should also support additional command line parameters used for topics, like ``--filter, --no-arr, --no-str, --include-msg-info``, etc.
```

Comment:
Since the command `ros2 service` does already exist, I would rephrase this to say we extend the existing tool with an echo verb.

Regarding style, keep the document to one sentence per line. This helps keep future diffs smaller.

Reply:
Done